piccolo/apps/migrations/auto/serialisation.py

```diff
@@ -521,12 +521,15 @@ def serialise_params(params: dict[str, Any]) -> SerialisedParams:
                 params=column._meta.params
             )
 
+            column_class_name = column.__class__.__name__
+            # we need to manually re-assign the base column
+            column._meta.name = f"{column_class_name.upper()}"
```

Comment:
There's no need for an f-string here.

This works fine:

```python
column._meta.name = column_class_name.upper()
```

Why do we have to set the column's name?

Reply:
I'm resetting `base_column` here because if we don't, `ValueError: `_name` is not defined - Table Metaclass should set it` is raised when we run the migration. If you have a better solution, please change this.